DependencyInjection/Configuration.php

```diff
@@ -72,8 +71,14 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
-            ->end()
-        ;
+                ->arrayNode('toolbar')
+                    ->canBeDisabled()
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('formatter')->defaultValue('httplug.collector.message_journal')->end()
```

Comment:
erm, i think this default value is wrong. i think it should defaultNull and if set it must be a service of a class that implements Http\Message\Formatter - or am i confused?